src/renderers/WebGLRenderer.js

```diff
@@ -2436,7 +2436,18 @@ class WebGLRenderer {
 
 		};
 
-		this.copyFramebufferToTexture = function ( position, texture, level = 0 ) {
+		this.copyFramebufferToTexture = function ( texture, position = null, level = 0 ) {
```

Comment:
If `position` is `null`, you have to make sure proper default value for `x` and `y` are used.

Right now, the method would through a null pointer exception if `position` would not be assigned to the method.

Reply:
make sense, I updated accordingly!